scripts/mixins/mctsmelteryio.zs

```diff
@@ -0,0 +1,24 @@
+#loader mixin
+
+import native.mezz.jei.api.recipe.IRecipeCategoryRegistration;
+import native.mezz.jei.api.IModRegistry;
+import native.mezz.jei.api.IJeiRuntime;
+
+#mixin Mixin
+#{targets: "mctmods.smelteryio.library.util.jei.JEI"}
```

Comment:
It feels like this tweaks not working and im still seeing categories in game ![](https://i.imgur.com/RReBNsn.gif)